awx/ui_next/src/components/JobList/JobListCancelButton.jsx

```diff
@@ -58,15 +58,29 @@ function JobListCancelButton({ i18n, jobsToCancel, onCancel }) {
                 one="You do not have permission to cancel the following job:"
                 other="You do not have permission to cancel the following jobs:"
               />
+              {cannotCancelPermissions.map((job, i) =>
+                i === cannotCancelPermissions.length - 1 ? (
+                  <strong> {job}</strong>
+                ) : (
+                  <strong> {job},</strong>
+                )
+              )}
```

Comment:
```suggestion
              {cannotCancelPermissions.join(',').map(text => <strong>{text}</strong>)}
```

You'll want to use [`join`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) for this kind of problem. 

![Screenshot from 2021-04-20 12-13-15](https://user-images.githubusercontent.com/9753817/115430383-50754980-a1d2-11eb-8da2-4f7cc1d48139.png)

Reply:
Actually, nevermind. This won't work for what we're doing here.